modules/oxide/src/less/theme/components/edit-area/edit-area.less

```diff
@@ -4,13 +4,25 @@
 
 @edit-area-iframe-background-color: @color-white;
 @edit-area-border: 0;
+@edit-area-border-color: #2D6ADF;
 
 .tox {
   .tox-edit-area {
     display: flex;
     flex: 1;
     overflow: hidden;
     position: relative; // Required for the iframe to resize in IE and Edge
+    &::before {
+      border: 2px solid @edit-area-border-color;
```

Comment:
I think these pixel values needs to be variables. Say you want to decrease the border with to 1px this is not currently possible with variable overrides.

Reply:
I added a variable for the border width.